website/docs/language/import/index.mdx

```diff
@@ -0,0 +1,130 @@
+---
+page_title: Import - Configuration Language
+description: >-
+  Import and manage existing resources with Terraform using configuration-driven import.
+---
+
+# Import
+
+-> **Note:** Import blocks are only available in Terraform v1.5.0 and later.
+
+~> **Experimental:** While we do not expect to make backwards-incompatible changes to syntax, the processing of imports during the plan stage, generating configuration, and the `-generate-config-out` flag may change in future releases.
+
+Use the `import` block to import existing infrastructure resources into Terraform, bringing them under Terraform's management. Configuration-driven import using `import` blocks is predictable, works with CICD pipelines, and allows you to preview an import operation before modifying state.
+
+Once imported, an existing resource is present in Terraform's configuration and state, much like any other resource Terraform created. You can then manage that resource like any other, updating its attributes and destroying it as part of a standard resource lifecycle.
```

Comment:
No, once import is complete the resource must be in configuration one way or another.
Either you used config generation, or you manually added a `resource` block alongside the `import` block.  

_Detail: Applying an import, like other planned actions (e.g. create), reconciles config and state. The CLI command `terraform import`, on the other hand, modifies state directly, allowing it to drift from config - it is more like the `state rm` and `state mv` state manipulation commands than a `plan` in that sense._
cc @rkoron007 FYI

If you try to import a resource without config and don't ask Terraform to generate it with the `-generate-config-out` flag, `terraform plan` will error:

```
$ terraform plan

Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Import block target does not exist
│ 
│   on main.tf line 1:
│    1: import {
│ 
│ The target for the given import block does not exist. If you wish to automatically generate config for this resource, use the
│ -generate-config-out option within terraform plan. Otherwise, make sure the target resource exists within your configuration. For example:
│ 
│   terraform plan -generate-config-out=generated.tf
╵
```